### PR TITLE
[FR] Update Beatmap Editor/Kiai Time

### PR DESCRIPTION
**Pull request by DoryanR**
Translation of the Kiai Time page

### CODE REVIEW

**Comment by almaloste** [comment]
Regarde la PR #3328 que tu as fermé, j'avais corrigé tout ce que je pouvais

**Comment by DoryanR** [comment]
C'est fait, hésite pas a regarder mes autres PR, même si je te l'accorde il y a beaucoup d'erreurs.

**Comment by DoryanR** [comment]
@MegaApplePi Check if the merge problem persists, I made some modifications, when I fixed the conflicts I made some mistakes, I will check the others which are problematic too.

File: wiki/Beatmap_Editor/Kiai_Time/fr.md [hunk 1]
```diff
@@ -1,21 +1,20 @@
-# Kiai Time
+# Kiai time
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor/Kiai_Time/fr.md]: same deal as https://github.com/ppy/osu-wiki/pull/3053#discussion_r375593804
```suggestion
---
no_native_review: true
---

# Kiai time
```

**Pull request merged**